integration/spark/src/main/scala/org/apache/carbondata/spark/util/CommonUtil.scala

```diff
@@ -122,14 +122,29 @@ object CommonUtil {
     var isValid: Boolean = true
     tableProperties.foreach {
       case (key, value) =>
-        if (!validateFields(key, fields)) {
+        if (!validateTblOption(key) || !validateFields(key, fields)) {
           isValid = false
           throw new MalformedCarbonCommandException(s"Invalid table properties ${ key }")
         }
     }
     isValid
   }
 
+  def validateTblOption(option: String): Boolean = {
+    val supportedOptions = Seq[String]("DICTIONARY_EXCLUDE", "DICTIONARY_INCLUDE", "COLUMN_GROUPS",
+      "TABLE_BLOCKSIZE", "NO_INVERTED_INDEX", "PARTITIONCLASS", "PARTITIONCOUNT",
```

Comment:
By this way, it is easy to forget to update this list if later more options are added. Better to refactory the options, like make it a Enum, and use it in both parser and here